### ISSUE TITLE
Stromversorgung über M-Bus

### ISSUE THREAD

**Issue by fabioaufinger** [tirolerstefan/kaifa#25]
Tolles Projekt. Ich bekomme auch bald einen Smart Meter, habe jedoch keinen vorbereiteten Stromanschluss im Verteilerkasten. Ist es auch möglich den RPi oder z.b. einen ESP32 über den M-Bus mit Strom zu versorgen?

**Comment by Phil1pp** [comment]
Nein, ist nicht möglich. Der M-Bus arbeitet mit sehr kleinen Strömen auf den Datenleitungen und bietet keine zusätzliche Spannungsversorgung.

**Comment by fabioaufinger** [comment]
Vielen Dank.
